pcdsdevices/lasers/elliptec.py

```diff
@@ -3,56 +3,137 @@
 """
 
 
-from ophyd import Component as Cpt
+from ophyd import FormattedComponent as FCpt
 from ophyd import Device
 from ophyd.signal import EpicsSignal, EpicsSignalRO
 
 
 class EllBase(Device):
     """
     Base class for Elliptec stages.
+    """
+
+    set_position = FCpt(EpicsSignal, '{prefix}:M{self._channel}:CURPOS',
```

Comment:
Not necessary to change, but FYI with recent versions of ophyd you no longer need the `self.` in formatted components:
```suggestion
    set_position = FCpt(EpicsSignal, '{prefix}:M{_channel}:CURPOS',
```

Reply:
so lightning does strike twice